packages/ui/src/styles/globals.css

```
@@ -118,6 +118,18 @@
   --color-sidebar-ring: var(--sidebar-ring);
 }
 
+@utility container {
+  margin-inline: auto;
+  width: 100%;
+  max-width: 80rem; /* max-w-7xl */
+  padding-inline: 1rem; /* px-4 */
+  @media (width >= 48rem) {
+    & {
+      padding-inline: 1.5rem; /* md:px-6 */
+    }
```

Stylelint (17.14.0)

[error] Redundant nested style rule (block-no-redundant-nested-style-rules)

(block-no-redundant-nested-style-rules)